pkg/controller/infinispan/resources/operatorconfig/monitoring_constants.go

```diff
@@ -0,0 +1,1684 @@
+package operatorconfig
+
+const dashboardJSON = `{
```

Comment:
Just idea for next PRs and issues: we could provide the ability to customize standard Grafana Dashboard json configuration by adding key to the ConfigMap and create it based on this value instead of hard-coded constant if needed

Reply:
+1
yeah, that could be a good enhancement